Program.cs

```diff
@@ -0,0 +1,52 @@
+﻿using Microsoft.AspNetCore.Identity;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Hosting;
+using Pomelo.EntityFrameworkCore.MySql.Infrastructure;
+using System;
+using System.Text;
+using Newtonsoft.Json;
+using Microsoft.AspNetCore.Diagnostics.HealthChecks;
+using Microsoft.Extensions.Diagnostics.HealthChecks;
+
+var builder = WebApplication.CreateBuilder(args);
+
+builder.Services.AddDbContext<ApplicationDbContext>(options =>
+{
+    var connectionString = "Server=localhost;Database=myappdb;User=myappuser;Password=mypassword;";

```

Comment:
- In General avoid constructing the connection string by hand. prone for bugs or typos.
- NEVER hard code secrets in code.
- You need to have this as part of you configuration. I can pair with you on this to make things go faster but finish the rest of the comments first.